rust/datafusion/src/scalar.rs

```diff
@@ -19,6 +19,11 @@
 
 use std::{convert::TryFrom, fmt, iter::repeat, sync::Arc};
 
+use arrow::array::{
```

Comment:
These imports don't seem to be necessary? The module already has `array::*,` ?

Reply:
Oh they were reformatted by `cargo fmt` which is what the README told me to run.....